client/snippets/snippets.json

```diff
@@ -364,5 +364,25 @@
         "prefix": "super",
         "body": "sendMessage(this, _CALL_ANCESTOR_OF, ${1:CURRENT_CLASS}, ${2:PARENT_CLASS}:::${3:PARENT_METHOD}, ${4:METHOD_ARGS});$0",
         "description": "Call a method of the parent class"
-    }
+    },
+	"If condition": {
+		"prefix": "\\if",
+		"body": [
+			"\\if(${1:condition})",
+			"\t$2",
+			"\\endIf"
+		],
+        "description": "“If” condition"
+	},
+	"If Else condition": {
+		"prefix": "\\if \\else",
+		"body": [
+			"\\if(${1:condition})",
+			"\t$2",
+			"\\else",
+			"\t$3",
+			"\\endIf"
+		],
+        "description": "“If/else” condition"
+	}
```

Comment:
You should update the list of snippets found in the readme. I think there's a command for that somewhere.